java/arcs/core/host/ArcHostContextParticle.kt

```diff
@@ -159,7 +160,7 @@ class ArcHostContextParticle(
         handles.arcHostContext.onReady { onReadyJobs["arcHostContext"]?.complete() }
         handles.handleConnections.onReady { onReadyJobs["handleConnections"]?.complete() }
         onReadyJobs.values.joinAll()
-        return withContext(coroutineContext) { block() }
+        return withContext(coroutineContext) { block() }.also { handleManager.close() }
```

Comment:
Why do we want to close the handle manager here?

Reply:
This handleManager is only used during serialization/deserialization or Arcs. It doesn't need to have a permanent connection all the time, only when Arcs are started/stopped.